### DISCUSSION TITLE
Welcome to OSINTBuddy!

### DISCUSSION THREAD

**Discussion by jerlendds** [jerlendds/osintbuddy#40] [Announcements]
## Welcome to OSINTBuddy!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what interests you about OSINT, this project, or data mining.

**Comment by LincolnKermit** [comment]
Hey guys, I want to create an website for osint recon, the project is still in developpement and I need one developper ( mostly back-end ) to help me go faster.
Project : https://github.com/LincolnKermit/FVP

**Reply by LivelyCarpet87** [reply]
This call seems rather open-ended. Could you please elaborate on the extent of the website, what you need, and what you currently see as your plan for implementation? In short, what is your stack and roadmap?

**Reply by LincolnKermit** [reply]
Im currently developping it in python for the api and flask lib for the back web.
I don't have any roadmap except it would be free ( except for the api )
Also, the front would be a template so we don't have to write the front.

**Reply by jerlendds** [reply]
Hey @LincolnKermit , looks like a cool project, I don't have the time to help with another project unfortunately or to vet some of the packages you're using which are unfamiliar to me but I wish you the best of luck with your project! Feel free to update us on any progress you make and you could also share this in the [OSINTBuddy Discord](https://discord.gg/gsbbYHA3K3) in the `#your-projects` channel if you want!

Good luck finding help!

**Comment by devcomputer** [comment]
is that discord channel still a thing? cant ready anything.
and like is there a roadmap when to expect that rewritten thing?

**Reply by jerlendds** [reply]
The discord is still a thing, yes! You'll be unable to read messages until someone is able to grant you the member role though. There is no public roadmap at the moment but feel free to reach out to me on discord if you have any specific questions or whatever else :)
